docs/modules/record-encryption/aws-kms/con-aws-kms-setup.adoc

```diff
@@ -91,9 +93,9 @@ EOF
 aws iam create-policy --policy-name KroxyliciousRecordEncryption --policy-document file:///tmp/policy
 ----
 
-== Apply Alias Based Policy to Filter Actor
+== Apply Alias Based Policy to Filter IAM Identity
 
-Attach the alias policy to the Filter Actor user.  This will allow the Filter actor to invoke the
+Attach the alias policy to the Filter IAM Identity.  This will allow the Filter to invoke the
```

Comment:
Similar thought to above `Filter's IAM Identity`. 

I'm not sure about capitalising the `F` in filter either...